CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+### PaymentSheet
+* [ADDED] Add `cvcRecollectionEnabledCallback` to `PaymentSheet.Builder` and `PaymentSheet.FlowController.Builder`
```

Comment:
I don't think we want to call this out as is. We will want to note it as experimental, and we probably want to keep it internal until we bug bash this.

Reply:
Removed, will add once bug bash is complete